### DISCUSSION TITLE
Insert Image

### DISCUSSION THREAD

**Discussion by olivierlopez44** [QtExcel/QXlsx#274] [Q&A]
hi Everybody,

I've got an issue when i want to insert and image in sheet.
The return of the function is true.
I can save the xlsx, but when i open it, i've got an error message:
"Sorry ... we have found a problem in ...."
So after there is nothing in a sheet where i have normally the image.
Any ideas ?

Thanks for the read and the help.

**Comment by j2doll** [comment]
Dear @olivierlopez44 

Sorry for late reply.

Can I see your test code?
